lang/ilias_ja.lang

```diff
@@ -12247,7 +12247,6 @@ rbac#:#cadm_edit_permission#:#User can change permission settings###23 12 2015 n
 common#:#no_gallery_users_available#:#There are currently no users in the gallery.###23 12 2015 new variable
 crs#:#crs_mem_contact#:#Tutorial Support###23 12 2015 new variable
 awrn#:#awareness_now_online#:#Online Now###23 12 2015 new variable
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]###23 12 2015 new variable

```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.